<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsWidgetGeneral</class>
 <widget class="QWidget" name="SettingsWidgetGeneral">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>456</width>
    <height>238</height>
   </rect>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
   </property>
   <item row="0" column="0">
    <widget class="QCheckBox" name="showTrayIconCheckBox">
     <property name="text">
      <string>Show system tray icon</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QCheckBox" name="rememberLastDatabasesCheckBox">
     <property name="text">
      <string>Remember last databases</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QCheckBox" name="openPreviousDatabasesOnStartupCheckBox">
     <property name="text">
      <string>Open previous databases on startup</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QCheckBox" name="modifiedExpandedChangedCheckBox">
     <property name="text">
      <string>Mark as modified on expanded state changes</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QCheckBox" name="autoSaveOnExitCheckBox">
     <property name="text">
      <string>Automatically save on exit</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QCheckBox" name="autoSaveAfterEveryChangeCheckBox">
     <property name="text">
      <string>Automatically save after every change</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Global Auto-Type shortcut</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="ShortcutWidget" name="autoTypeShortcutWidget"/>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="hideToTrayWhenMinimizingCheckBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Minimizing window hides to tray</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QCheckBox" name="hideToTrayWhenClosingCheckBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Closing window hides to tray</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ShortcutWidget</class>
   <extends>QLineEdit</extends>
   <header>autotype/ShortcutWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
